### PR TITLE
label and button component plus reset css

### PR DESCRIPTION
**Pull request by lubomir1618**

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import "./Button.css";
+
+export interface Props {
+  value: string;
+}
+
+export function dummy() {
```
Comment (tomcosk) [inline src/components/button/Button.tsx]:
```suggestion
export function handleClick() {
```

File: src/components/button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  return;
+}
+
+export const Button = (props: Props) => (
+  <div className="button" onClick={dummy}>
```
Comment (tomcosk) [inline src/components/button/Button.tsx]:
```suggestion
  <div className="button" onClick={handleClick}>
```

File: src/components/label/Label.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import "./Label.css";
+
+export interface Props {
+  type: string;
+  value: number | string;
+}
+
+export function pluralize(word: string, value: number | string): string {
```
Comment (tomcosk) [inline src/components/label/Label.tsx]: je to vseobecna funkcia, ktora moze byt pouzita hocikde inde, odporucam presunut do nejakeho utils.ts niekde

Comment (lubomir1618) [reply src/components/label/Label.tsx]: done

File: src/components/button/Button.tsx (hunk 2 above)
Comment (lubomir1618) [reply src/components/button/Button.tsx]: done

File: src/components/button/Button.tsx (hunk 1 above)
Comment (lubomir1618) [reply src/components/button/Button.tsx]: done

File: src/components/button/Button.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+:root {
+  --text-color-label: #6eb5fc;
+}
+
+.button {
```
Comment (lukasorgovan) [inline src/components/button/Button.css]: the button css should be named as Button.module.css and should be loaded to the component as in example test component (in this app we will try to use the module css system (which should be completely isolated from the outer world) - same applies for other css files in this pull request.

**Pull request merged**